packages/protocol/runTests.js

```diff
@@ -72,7 +72,9 @@ async function test() {
 
     const testGlob =
       argv._.length > 0
-        ? argv._.map((testName) => `test/\*\*/${testName}.ts`).join(' ')
+        ? argv._.map((testName) =>
+            testName.endsWith('/') ? `test/${testName}*.ts` : `test/\*\*/${testName}.ts`
```

Comment:
Mmm, check this line.
Probably is `test/${testName}**/*.ts`
Otherwise you won't be running subfolders

Reply:
> Just looking at the GitHub UI, doesn't look like these checks are required to pass for a PR to merge, which I believe they should?

Great catch. I'll look into how that's configured